Source/CsDebugScript.Engine/Engine/Debuggers/DbgEngDll.cs

```diff
@@ -1041,6 +1041,69 @@ public bool IsMinidump(Process process)
             return false;
         }
 
+       /// <summary>
+        /// Gets Lasts Event, most exception or event.
+        /// </summary>
+        /// <returns></returns>
+        public DebugEventInfo GetLastEventInfo()
+        {
+            uint type;
+            uint processId;
+            uint threadId;
+            uint extraInfoUsed;
+            uint descriptionSize;
+
+            //  Collect EventSize 
```

Comment:
Use single line comments with one space:
// Comment
No need for second empty comment line